mythril/laser/ethereum/svm.py

```diff
@@ -78,7 +78,7 @@ def __init__(
         self.work_list = []  # type: List[GlobalState]
         self.strategy = strategy(self.work_list, max_depth)
         self.max_depth = max_depth
-        self.transaction_count = transaction_count
+        self.transaction_count = transaction_count or 1
```

Comment:
there is already a default value right. I don't think this should be fixed here, but on a higher level